### ISSUE TITLE
storage/engine: surface compaction health in admin UI

### ISSUE THREAD

**Issue by dt** [cockroachdb/cockroach#41265]
TL;DR: hook up time series recording of L0 file count and estimated_pending_compaction_bytes.

In debugging bulk-ingestion slowdowns, we eventually figured out that lots of out ingestions were being back pressured due to L0 file count, however beyond that, that on the node where L0 was filling up, files were going to L0 because compaction was unable to keep up and actually empty the lower levels.

One thing that might have made it easier to diagnose would have been time-series data revealing the climbing compaction debt (`estimated_pending_compaction_bytes`) on the node that eventually became the problem, as well as the L0 file count that actually caused the back pressure. Being able to see these correlated with the other metrics might have helped us zero in on it sooner.

Interval compaction bytes would also be nice.

**Comment by dt** [comment]
I actually went ahead and added an `estimated_pending_compaction_bytes` time series #41754 though it isn't currently in any default UIs (e.g. the Storage dashboard).

interval compaction would be still be nice, even if just for debugging use in custom charts.

**Comment by dt** [comment]
the existing "read-amplification" chart already serves as a fine L0 count graph (when just read with a constant -5).
